packages/wonder-blocks-breadcrumb/components/breadcrumb.test.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import * as React from "react";
+import {shallow} from "enzyme";
+
+import Breadcrumb from "./breadcrumb.js";
+import BreadcrumbItem from "./breadcrumb-item.js";
+
+describe("Breadcrumb", () => {
+    it("should render a Breadcrumb", () => {
```

Comment:
Not sure about adding this test here. Might be a bit unnecessary and it's covered by the auto-generated snapshot test.